sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -74,7 +86,36 @@ internal class RadarApiClient(
         )
     }
 
-    internal fun getConfig() {
+    data class RadarMeta(
+        val config: JSONObject?,
+        val remoteTrackingOptions: RadarTrackingOptions?
+    )
+
+    private fun parseMeta(res: JSONObject?): RadarMeta {
+        var meta: JSONObject? = null
+        if (res?.has("meta") == true) {
+            meta = res.getJSONObject("meta")
+        }
+
+        var config: JSONObject? = null
+        if (meta?.has("config") == true) {
+            config = meta.getJSONObject("config")
+        }
+
+        var remoteTrackingOptions: RadarTrackingOptions? = null
+        if (meta?.has("trackingOptions") == true) {
+            try {
+                val rawOptions = meta.getJSONObject("trackingOptions")
+                remoteTrackingOptions = RadarTrackingOptions.fromJson(rawOptions)
+            } catch (e: Exception) {
```

Comment:
Can you use a more explicit error here? Would it be a `JSONException`?

Reply:
I think this is actually coded in a way that wouldn't throw... so I might just take this out.